changelog/fragments/sdk-fbc-run-bundle(-upgrade).yaml

```diff
@@ -0,0 +1,19 @@
+entries:
+  - description: >
+      Add File-Based Catalog support for both `run bundle` and `run bundle-upgrade` subcommands. 
+      Infer the image type and handle FBC scenarios accordingly for both commands. For FBC images,
+      Operator Registry APIs are called for rendering the bundle and/or the index image and serving these images
+      through the registry pod using a gRPC port. For images that are SQLite based, 
+      `opm registry add` and `opm serve` commands will be called during the registry pod creation. 
+      Support for these old opm commands will be deprecated and completely removed in the future. 
```

Comment:
> Add support to [file-based-catalog](https://olm.operatorframework.io/docs/reference/file-based-catalogs/) for the subcommands [operator-sdk run bundle](add the link for the docs) and [run bundle-upgrade] so that new indexes created by these subcommands are using the new format as users are able to inform an index catalog with FBC format via the flag option `--index-image`.

Reply:
WDYT?

Reply:
I like what @camilamacedo86 wrote EXCEPT the to and for are flipped. 

> Add support for [file-based-catalog](https://olm.operatorframework.io/docs/reference/file-based-catalogs/) to the subcommands [operator-sdk run bundle](add the link for the docs) and [run bundle-upgrade] so that new indexes created by these subcommands are using the new format. Users are able to pass in an index catalog with FBC format via the flag option --index-image.

Reply:
Fixed the changelog based on the suggestions.